features/frameworks/mbed-coap/source/sn_coap_protocol.c

```diff
@@ -1203,6 +1203,8 @@ void sn_coap_protocol_linked_list_duplication_info_remove(struct coap_s *handle,
     (void)msg_id;
 #endif //SN_COAP_DUPLICATION_MAX_MSGS_COUNT
 }
+
+#if SN_COAP_DUPLICATION_MAX_MSGS_COUNT
```

Comment:
Mbed Coap comes from external source.
@mikter or @artokin Is this acceptable to be modified here?

Reply:
> @mikter or @artokin Is this acceptable to be modified here?

Any update? Can this go in or not?

Reply:
All other references to sn_coap_protocol_duplication_info_free are already flagged by SN_COAP_DUPLICATION_MAX_MSGS_COUNT and therefore this is good to go.

CC: @anttiylitokola